### DISCUSSION TITLE
Set up error: Server with name 'LibreELEC' is not defined

### DISCUSSION THREAD

**Discussion by ChuckyNorris** [Taxel/PlexTraktSync#1860] [Support] [answered]
I'm setting up PlexTraktSync (in docker) on my RPi400 running LibreElec with a Plex server (also running in docker). I get through most of the set up, including logging into Plex, connecting to the server and logging into the Trakt API, but then I get:

`ERROR    Server with name 'LibreELEC' is not defined`

Any idea what could be going wrong?

**Comment by glensc** [comment, marked as answer]
read [discussions guide](https://github.com/Taxel/PlexTraktSync/discussions/803):
> If You have application crash or error, then those must be reported as [issues](https://github.com/Taxel/PlexTraktSync/issues/new/choose)❗
